packages/lesswrong/lib/collections/posts/views.ts

```diff
@@ -1295,13 +1316,6 @@ Posts.addView("personalTagProgressPosts", (terms: PostsViewTerms) => {
   }
 })
 
-ensureIndex(Posts,
-  augmentForDefaultView({ userId: 1, tagRelevance: 1}),
-  {
-    name: "posts.users_tagged_posts",
```

Comment:
This had been used for the tagging dashboard to help users identify their own posts that hadn't been tagged (so they could take responsibility for tagging them). We're not currently using it.